packages/components/avatar/src/avatar.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import Avatar from './avatar';
+import { iconArgType } from '@/storybook-helpers';
+
+const meta: Meta<typeof Avatar> = {
+  title: 'components/Avatar',
+  component: Avatar,
+  argTypes: {
+    icon: iconArgType,
+  },
+};
+export default meta;
+
+type Story = StoryObj<typeof Avatar>;
+
+export const BasicExample: Story = {
```

Comment:
This syntax is _much easier to implement_, what an improvement over v5!

Reply:
What I also noticed is, that if you use `StoryObj` instead of `StoryFn`, you can use the Storybook-UI to create new stories after changing the controls (lower right corner: Save, New, Reset):

![image](https://github.com/user-attachments/assets/e4242f09-c9a2-464c-bc05-971d67c96ef4)